components/main-chef-wrapper/lib/version.go

```diff
@@ -123,11 +123,18 @@ func omnibusRoot()string  {
 }
 
 func ExpectedOmnibusRoot()string {
-	groot := os.Getenv("GEM_ROOT")
-	rootPath, err := filepath.Abs(path.Join(groot,"..","..", "..", "..", ".."))
+	ex, _ := os.Executable()
+	exReal, err := filepath.EvalSymlinks(ex)
 	if err != nil {
 		fmt.Fprintln(os.Stderr, "ERROR:", err)
 	}
+	rootPath := filepath.Dir(exReal)
```

Comment:
So this should give us `/opt/chef-workstation/bin` or `c:/opscode/chef-workstation/bin`. In both cases we want it taken up a level, to the `chef-workstation` dir.